.stickler.yml

```diff
@@ -0,0 +1,15 @@
+# add the linters you want stickler to use for this project
+linters:
+  eslint:
+    # indicate where is the config file for stylelint
+    config: '.eslintrc.yml'
+
+# add the files here you want to be ignored by stylelint
+files:
+  ignore:
+      - 'dist/*'
+
+# PLEASE DO NOT enable auto fixing options
+# if you need extra support from you linter - do it in your local env as described in README for this config
+
+# find full documentation here: https://stickler-ci.com/docs
```

Comment:
This file is different from the config file provided by microverse
Please [follow instructions here](https://github.com/microverseinc/linters-config/tree/master/javascript) and use linting provided for you